util/importers/LDC97S62.py

```diff
@@ -1,39 +1,35 @@
-from __future__ import print_function
-from __future__ import absolute_import
+from __future__ import absolute_import, division, print_function
+
 import fnmatch
-import numpy as np
 import os
+import pandas
 import subprocess
-import wave
 import tensorflow as tf
 import unicodedata
-import codecs
-from glob import glob
-from itertools import cycle
+import wave
+
 from math import ceil
-from Queue import PriorityQueue
-from Queue import Queue
+from six.moves import range
 from threading import Thread
 from util.audio import audiofile_to_input_vector
-from util.gpu import get_available_gpus
 from util.data_set_helpers import DataSets
+from util.gpu import get_available_gpus
 from util.text import text_to_char_array, ctc_label_dense_to_sparse, validate_label
-from six.moves import range
 
 class DataSet(object):
-    def __init__(self, txt_files, thread_count, batch_size, numcep, numcontext, next_index=lambda x: x + 1):
+    def __init__(self, filelist, thread_count, batch_size, numcep, numcontext, next_index=lambda x: x + 1):
         self._coord = None
         self._numcep = numcep
         self._x = tf.placeholder(tf.float32, [None, numcep + (2 * numcep * numcontext)])
         self._x_length = tf.placeholder(tf.int32, [])
-        self._y = tf.placeholder(tf.int32, [None, ])
+        self._y = tf.placeholder(tf.int32, [None,])
         self._y_length = tf.placeholder(tf.int32, [])
         self.example_queue = tf.PaddingFIFOQueue(shapes=[[None, numcep + (2 * numcep * numcontext)], [], [None, ], []],
                                                   dtypes = [tf.float32, tf.int32, tf.int32, tf.int32],
                                                   capacity = 2 * self._get_device_count() * batch_size)
         self._enqueue_op = self.example_queue.enqueue([self._x, self._x_length, self._y, self._y_length])
         self._close_op = self.example_queue.close(cancel_pending_enqueues=True)
-        self._txt_files = txt_files
+        self._filelist = filelist
```

Comment:
Maybe remove this member? Having both the name `_filelist` and `_files_list` is a bit confusing.

An idea would be to 
- Remove `_filelist` completely
- Pass `filelist` to `_create_files_list()`
- Use `_files_list` in `total_batches()`